### ISSUE TITLE
RecoverFolder no longer working

### ISSUE THREAD

**Issue by darchon7** [sandboxie-plus/Sandboxie#1254]
**Describe the bug**
files downloaded to my RecoverFolder used to go straight out of the sandbox into the actual folder but now no longer works and a "DefaultBox - File Recovery" now appears every time requiring me to click Recover.

**To Reproduce**
1. download a file
2. it prompts for a location
3. "DefaultBox - File Recovery" appears
4. click Recover button

**Expected behavior**
it should immediately download to my actual folder outside the sandbox and no File Recovery window should appear.

**Screenshots**
n/a

**System details and installed software (please provide the following information):**
 - What is your Windows edition and version?  Windows 10 Pro 20H2
 - What is your current Sandboxie edition and version?  Sandboxie Plus 0.9.6
 - Please mention in which version this bug got introduced, because it acts as a guide for testers and developers. 0.9.6
 - Please mention any security software running in the background, even if the real-time protection is disabled.
 - Bitdefender

**Additional context**
no

**Sandboxie configuration**

[GlobalSettings]

Template=RpcPortBindings
Template=WindowsRasMan
Template=BitDefenderInternetSecurity
Template=7zipShellEx
Template=Listary
Template=WindowsLive
Template=OfficeLicensing
Template=OfficeClickToRun
Template=RoboForm
FileRootPath=d:\Sandbox\%USER%\%SANDBOX%
KnownConflicts=BitDefender
SeparateUserFolders=y
KeyRootPath=\REGISTRY\USER\Sandbox_%USER%_%SANDBOX%
IpcRootPath=\Sandbox\%USER%\%SANDBOX%\Session_%SESSION%
EditAdminOnly=n
ForceDisableAdminOnly=n
ForgetPassword=n

[DefaultBox]

ConfigLevel=9
AutoRecover=y
BlockNetworkFiles=n
Template=OpenBluetooth
Template=SkipHook
Template=FileCopy
Template=qWave
Template=BlockPorts
Template=LingerPrograms
Template=Chrome_Phishing_DirectAccess
Template=Firefox_Phishing_DirectAccess
Template=AutoRecoverIgnore
RecoverFolder=%{374DE290-123F-4565-9164-39C4925E467B}%
RecoverFolder=%Personal%
RecoverFolder=%Favorites%
RecoverFolder=%Desktop%
RecoverFolder=d:\Program Files
RecoverFolder=F:\Downloads
RecoverFolder=C:\Users\dwayne\Downloads
BorderColor=#00ffff,ttl,6
Enabled=y
OpenFilePath=<firefoxprograms>,D:\Program Files\
OpenFilePath=firefoxportable.exe,D:\Program Files\
OpenFilePath=firefox.exe,D:\Program Files\
OpenFilePath=<firefoxprograms>,F:\Downloads\
OpenFilePath=<firefoxprograms>,D:\Program Files\
OpenFilePath=robotaskbaricon.exe,%Personal%\My RoboForm Data\
OpenFilePath=robotaskbaricon.exe,C:\Program Files (x86)\Siber Systems\AI RoboForm\
OpenFilePath=qbittorrent.exe,%AppData%\qBittorrent\
OpenFilePath=qbittorrent.exe,%Local AppData%\qBittorrent\
OpenFilePath=qbittorrent.exe,H:\
OpenFilePath=operaportable.exe,D:\Program Files\
OpenFilePath=opera.exe,D:\Program Files\
OpenFilePath=chrome.exe,D:\Program Files\
OpenFilePath=googlechromeportable.exe,D:\Program Files\
OpenPipePath=qbittorrent.exe,%{374DE290-123F-4565-9164-39C4925E467B}%\
ClosedFilePath=*BitDefender*atcuf*.dll
WriteFilePath=C:\Program Files\Bitdefender\*
OpenWinClass=#
BoxNameTitle=n
BlockNetParam=y
DropAdminRights=n
FakeAdminRights=n
ClosePrintSpooler=n
OpenPrintSpooler=n
AllowSpoolerPrintToFile=n
CopyLimitKb=81920
CopyLimitSilent=n
NeverDelete=n
AutoDelete=n
AllowRawDiskRead=n
NotifyDirectDiskAccess=n

[UserSettings_11C20289]

SbieCtrl_UserName=dwayne
SbieCtrl_BoxExpandedView=DefaultBox
SbieCtrl_NextUpdateCheck=1614796928
SbieCtrl_WindowCoords=200,150,909,596
SbieCtrl_ActiveView=40021
SbieCtrl_UpdateCheckNotify=n
SbieCtrl_AutoApplySettings=n
SbieCtrl_ProcessViewColumnWidths=250,70,300
SbieCtrl_AutoStartAgent=SandMan.exe
SbieCtrl_EnableAutoStart=y

[UserSettings]

SbieCtrl_AutoStartAgent=SandMan.exe
AutoRecover=n
BlockNetworkFiles=y
RecoverFolder=%{374DE290-123F-4565-9164-39C4925E467B}%
RecoverFolder=%Personal%
RecoverFolder=%Desktop%
BorderColor=#00FFFF,ttl
Template=OpenBluetooth
Template=SkipHook
Template=FileCopy
Template=qWave
Template=BlockPorts
Template=LingerPrograms
Template=Chrome_Phishing_DirectAccess
Template=Firefox_Phishing_DirectAccess
Template=AutoRecoverIgnore
ConfigLevel=9

**Comment by DavidXanatos** [comment]
Did you have earlier an open path configured?

**Comment by darchon7** [comment]
yes this is the exact same config I've been using all along. I have not made any changes.

**Comment by darchon7** [comment]
ohh sorry the editor cropped out the < > values, it should really read:

`OpenFilePath=<firefoxprograms>,D:\Program Files\`
`OpenFilePath=<firefoxprograms>,F:\Downloads\`
`OpenFilePath=<firefoxprograms>,D:\Program Files\`
